Source/WebKit/Shared/WTFArgumentCoders.serialization.in

```diff
@@ -52,6 +54,141 @@ header: <wtf/text/AtomString.h>
+template: enum class WebCore::FetchIdentifierType
+template: enum class WebCore::ImageDecoderIdentifierType
+template: enum class WebCore::ImageOverlayDataDetectionResultIdentifierType
+template: enum class WebCore::InbandGenericCueIdentifierType
+template: enum class WebCore::LayerHostingContextIdentifierType
+template: enum class WebCore::MediaSessionGroupIdentifierType
+template: enum class WebCore::MediaUniqueIdentifierType
+template: enum class WebCore::ProcessIdentifierType
+template: enum class WebCore::PushSubscriptionIdentifierType
+template: enum class WebCore::RealtimeMediaSourceIdentifierType
+template: enum class WebCore::SWServerToContextConnectionIdentifierType
+template: enum class WebCore::SharedWorkerIdentifierType
+template: enum class WebCore::SpeechRecognitionConnectionClientIdentifierType
+template: enum class WebCore::TextCheckingRequestIdentifierType
+template: enum class WebCore::TextManipulationItemIdentifierType
+template: enum class WebCore::TextManipulationTokenIdentifierType
+template: enum class WebCore::WindowIdentifierType
+template: enum class WebCore::WorkletGlobalScopeIdentifierType
+template: enum class WebKit::AudioMediaStreamTrackRendererInternalUnitIdentifierType
+template: enum class WebKit::ContentWorldIdentifierType
+template: enum class WebKit::DownloadIdentifierType
+template: enum class WebKit::ImageAnalysisRequestIdentifierType
+template: enum class WebKit::MediaDevicePermissionRequestIdentifierType
+template: enum class WebKit::MediaRecorderIdentifierType
+template: enum class WebKit::RemoteAudioHardwareListenerIdentifierType
+template: enum class WebKit::RemoteAudioSessionIdentifierType
+template: enum class WebKit::RemoteCDMIdentifierType
+template: enum class WebKit::RemoteCDMInstanceIdentifierType
+template: enum class WebKit::RemoteCDMInstanceSessionIdentifierType
+template: enum class WebKit::RemoteLegacyCDMIdentifierType
+template: enum class WebKit::RemoteLegacyCDMSessionIdentifierType
+template: enum class WebKit::RemoteMediaResourceIdentifierType
+template: enum class WebKit::RemoteMediaSourceIdentifierType
+template: enum class WebKit::RemoteRemoteCommandListenerIdentifierType
+template: enum class WebKit::RemoteSourceBufferIdentifierType
+template: enum class WebKit::RenderingBackendIdentifierType
+template: enum class WebKit::RetrieveRecordResponseBodyCallbackIdentifierType
+template: enum class WebKit::SampleBufferDisplayLayerIdentifierType
+template: enum class WebKit::ShapeDetectionIdentifierType
+template: enum class WebKit::StorageAreaImplIdentifierType
+template: enum class WebKit::StorageAreaMapIdentifierType
+template: enum class WebKit::StorageNamespaceIdentifierType
+template: enum class WebKit::TrackPrivateRemoteIdentifierType
+template: enum class WebKit::WCLayerTreeHostIdentifierType
+template: enum class WebKit::WebExtensionControllerIdentifierType
+template: enum class WebKit::WebGPUIdentifierType
+template: enum class WebKit::WebsiteDataType : uint32_t
```

Comment:
I don't see ObjectIdentifier<WebKit::WebsiteDataType> or ObjectIdentifier<WebsiteDataType> anywhere.  Is this really needed?  (along with the special parsing logic)

Reply:
I only added ones that needed to be there for the build.

Will double check.

Reply:
Was, indeed, not needed.

Removed, but leaving the storage type logic + its test, because it's gonna be relevant soon.